controllers/lvmcluster_controller_watches.go

```diff
@@ -44,6 +45,10 @@ func (r *LVMClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			&storagev1.StorageClass{},
 			handler.EnqueueRequestsFromMapFunc(r.getLVMClusterObjsForReconcile),
 		).
+		Watches(
```

Comment:
Why do we add a node watcher to lvmcluster controller?

Reply:
This is because of updateLVMClusterStatus() which removes the vg from the NodeStatus in the LVMCluster if the node is removed. Otherwise it would be orphaned in the Status

Reply:
NVM just saw we also watch NodeStatus... removed it.